### ISSUE TITLE
Missing #include <string> in src/gui/dialogs/browser/browserBase.h

### ISSUE THREAD

**Issue by musicinmybrain** [monocasual/giada#454]
**Environment**
- OS: Fedora Rawhide (35)
- Giada version: 0.17.2

**Describe the bug**
Fails to compile with errors like:

```
In file included from ../src/gui/dialogs/browser/browserDir.h:30,
                 from ../src/gui/dialogs/browser/browserDir.cpp:27:
../src/gui/dialogs/browser/browserBase.h:57:14: error: 'string' in namespace 'std' does not name a type
   57 |         std::string getSelectedItem() const;
      |              ^~~~~~
```

**To Reproduce**
1. Use a Fedora 35/Rawhide VM (or something with the same g++ and standard library versions).
2. Compile normally.

Or just look at the source to see the missing include.

**Expected behavior**
No compiler errors.

**Screenshots**
N/A

**Additional context**
The file `src/gui/dialogs/browser/browserBase.h` uses `std::string`, so it should `#include <string>` directly rather than relying on an indirect include via some other header. I have confirmed that this fixes the problem.

**Comment by terencode** [comment]
6559aa56ac95cc7760d399b5a624505c5f6a4df1 reintroduced the problem on a different file:

```
In file included from src/gui/elems/basics/input.cpp:27:
src/gui/elems/basics/input.h:38:39: error: ‘string’ in namespace ‘std’ does not name a type
   38 |         std::function<void(const std::string&)> onChange = nullptr;
      |                                       ^~~~~~
src/gui/elems/basics/input.h:32:1: note: ‘std::string’ is defined in header ‘<string>’; did you forget to ‘#include <string>’?
   31 | #include <functional>
  +++ |+#include <string>
   32 | 
````

**Comment by gvnnz** [comment]
@terencode fixed in 61c8d6efab7111e85283f9f78353667a9282deca, thanks!
